spec/index.adoc

```diff
@@ -438,6 +438,13 @@ If the canister has a https://webassembly.github.io/spec/core/binary/modules.htm
 +
 It is recommended for the canister to have a custom section called "icp:public candid:service", which contains the UTF-8 encoding of https://github.com/dfinity/candid/blob/master/spec/Candid.md#core-grammar[the Candid interface] for the canister.
 
+* `/canister/<canister_id>/property/<name>` (blob):
++
+Users can set arbitrary canister properties via settings, see <<ic-create_canister>> and <<ic-update_settings>>. The (blob) is optional. There is a maximum combined size limit for all properties of 4KB. Properties may be used to enable or disable system features, e.g.:
+* disable_ic0_app_service_worker
+* allow_internet_identity_aliases where (blob) is a comma separated of canister ids and/or domains.
```

Comment:
Editorial comment: The Spec has no notion about service workers or Internet Identity, and so far we try to keep the document (mostly) concise and to the point. So maybe move the examples into a Note? (although my suggested edit above shrinks this anyways)